test/clojure/core_test/double.cljc

```diff
@@ -23,6 +23,7 @@
        ;; In cljs, `double` is just returns the argument unchanged (dummy fn)
        [(is (= "0" (double "0")))
         (is (= :0 (double :0)))]
+       :jank []
```

Comment:
What does jank do for these two cases? We can't just not test these two cases with jank.